static/src/stylesheets/layout/new-header/_menu-item.scss

```diff
@@ -0,0 +1,127 @@
+.menu-item {
+    position: relative;
+    width: 100%;
+
+    > *[data-link-name='nav2 : the guardian app'],
```

Comment:
My first reflex about adding classnames was the same, but then I thought it's actually a good way of saying: hey - something special is going on here. If we don't do it here, we'd have to hide the ugly logic for doing this in Scala and so, in the sense of "move complexity to the edge" I'm in a favor of keeping it. Also: we won't see this in the desktop design, so the classes would be useless there ...